Customization/T210/PhoneRepairShop_Code/PhoneRepairShop_Code/RSSVRepairPriceMaint.cs

```diff
@@ -72,15 +72,15 @@ protected void _(Events.FieldDefaulting<RSSVRepairItem,
             if (row.InventoryID != null)
             {
                 //Use the PXSelector attribute to select the stock item.
-                var item = PXSelectorAttribute.
+                InventoryItem item = PXSelectorAttribute.
                     Select<RSSVRepairItem.inventoryID>(e.Cache, row)
                     as InventoryItem;
                 //Retrieve the base price for the stock item.
-                var curySettings =
+                InventoryItemCurySettings curySettings =
                     InventoryItemCurySettings.PK.Find(
-                    this, item?.InventoryID, Accessinfo.BaseCuryID ?? "USD");
+                    this, item.InventoryID, Accessinfo.BaseCuryID ?? "USD");
                 //Copy the base price from the stock item to the row.
-                if (curySettings != null) e.NewValue = curySettings.BasePrice;
+                e.NewValue = curySettings.BasePrice;

```

Comment:
Changes in this file override null reference feature changes. Please revert and make sure the course contains proper instructions.